src/components/Banner/Banner.tsx

```diff
@@ -76,12 +79,19 @@ const BannerControls = ({
 
   return (
     <ControlsContainer>
-      <ChevronIcon onClick={handleLeftChevronClick} disabled={isFirstBanner} />
+      <span onClick={handleLeftChevronClick}>
+        <Icon.Chevron
+          direction={IconDirection.Left}
+          // disabled={isFirstBanner}
```

Comment:
Let's add a "TODO" here and explain why this has been commented out please, I like the idea of the `IconButton` replacing this `span` in the near future. We could leave that documented here in the code for now

Reply:
Gotcha will add a TODO

Reply:
resolved [fix Direction.ts, capitalize Enums, add TODO on banner component](https://github.com/Major-Eth/yearn-web-lib/pull/21/commits/df32714bb98ffa32befd9f149a24aca802303c21)